### PR TITLE
fix: auto generated types

### PR DESCRIPTION
**Pull request by DemonHa**

### CODE REVIEW

**Comment by DemonHa** [comment]
Also one more thing, doesn't it make more sense for this auto generated file to be .d.ts?

**Comment by DemonHa** [comment]
@taozhi8833998  Can you have a look at this PR?

**Comment by taozhi8833998** [comment]
@DemonHa I will review it today, Sorry for the late review.

File: pegjs/postgresql.pegjs [hunk 1]
```diff
@@ -3187,7 +3187,15 @@ table_name
 
 or_and_expr
 	= head:expr tail:(__ (KW_AND / KW_OR) __ expr)* {
-    // => binary_expr
+    /*
+    export type binary_expr = {
+      type: 'binary_expr';
+      operator: string;
+      left: string;
+      right: string;
```
Comment (taozhi8833998) [inline pegjs/postgresql.pegjs:3195]: @DemonHa left and right property should be expr? and we have binary_expr before, do not change it here.

Comment (taozhi8833998) [reply pegjs/postgresql.pegjs:3195]: the postgresql.ts before
```javascript
export type BINARY_OPERATORS = LOGIC_OPERATOR | 'OR' | 'AND' | multiplicative_operator | additive_operator
      | arithmetic_comparison_operator
      | 'IN' | 'NOT IN'
      | 'BETWEEN' | 'NOT BETWEEN'
      | 'IS' | 'IS NOT'
      | 'LIKE'
      | '@>' | '<@' | OPERATOR_CONCATENATION | DOUBLE_WELL_ARROW | WELL_ARROW | '?' | '?|' | '?&' | '#-'
    export interface binary_expr {
      type: 'binary_expr',
      operator: BINARY_OPERATORS,
      left: expr,
      right: expr
    }
```

**Comment by taozhi8833998** [comment]
> Also one more thing, doesn't it make more sense for this auto generated file to be .d.ts?

yes, you are right, I will make it happen in the future.

File: pegjs/postgresql.pegjs (hunk 1 above)
Comment (DemonHa) [reply pegjs/postgresql.pegjs:3195]: Ok, I see it now, but I think the typegen output file was modified by hand, I will change it!

**Comment by DemonHa** [comment]
@taozhi8833998 Can you have another look at the changes?

**Comment by taozhi8833998** [comment]
@DemonHa Thanks for the great pr, merge it.

**Pull request merged**